src/Login/Login.jsx

```diff
@@ -139,7 +139,7 @@ class Login extends React.Component {
                       isClearable
                       isSearchable
                       options={loginOptions[i]}
-                      filterOption={createFilter({ ignoreAccents: true, ignoreCase: true })}
+                      filterOption={createFilter({ ignoreAccents: false, matchFrom: 'any', stringify: option => `${option.label}` })}
```

Comment:
you said `ignoreAccents: true` was slowing down the search, i assume `ignoreCase: true` was too?

what do `matchFrom: 'any'` and `stringify` do? i can't find docs :(

Reply:
I can't find docs on `stringify` either 😢 but I found it on a gh issue for searching over just labels and not values: https://github.com/JedWatson/react-select/issues/3403

I think this is the closest to docs on the createFilter function: https://react-select.com/advanced#custom-filter-logic

ignoreCase: true is actually the default to my understanding, I think it's also slowing down search but I found it more confusing to do case-sensitive search 🤷 If perf is an issue we can start there and then stick react-windowed back in if we need to.

Reply:
i realize i'm being annoying but we might want to add a comment in the code to explain what these parameters do, since it's not documented (🤦‍♀️) and i agree the search should not be case sensitive

Reply:
Not at all, that's a great point! Lemme stick a comment in there